games-util/springlobby/springlobby-0.264.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+WX_GTK_VER="3.0"
+inherit cmake-utils eutils flag-o-matic wxwidgets
+
+DESCRIPTION="The official lobby client for SpringRTS community games"
+HOMEPAGE="https://springlobby.info"
+SRC_URI="https://www.springlobby.info/tarballs/${P}.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+RESTRICT="mirror"
+IUSE="debug +libnotify +nls +sound"
+
+RDEPEND="
+	>=dev-libs/boost-1.35:=
+	dev-libs/openssl:*
+	sound? ( media-libs/alure
+		media-libs/openal
+	)
+	net-misc/curl
+	sys-libs/zlib[minizip]
+	x11-libs/libICE
+	x11-libs/libSM
+	x11-libs/libXext
+	x11-libs/wxGTK:${WX_GTK_VER}[X]
+	libnotify? ( x11-libs/libnotify )
+"
+
+DEPEND="${RDEPEND}
+	nls? ( sys-devel/gettext )
+"
+
+pkg_setup() {
+	setup-wxwidgets
```

Comment:
Move this to `src_configure`. `pkg_setup` gets executed for binary packages where this isn't necessary.

Reply:
fixed